<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchedulerAnalysis</class>
 <widget class="QDialog" name="SchedulerAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scheduler Analysis</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="noticeLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Note that changes here will not be saved in the project, this tool is strictly read-only and for analysis purposes</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="topLayout" stretch="1,0">
     <item>
      <widget class="QTableView" name="taskTable"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="inspectionLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labelInspectTimeSlot">
           <property name="text">
            <string>Inspect timeslot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="currentTimeSlotSpinBox">
           <property name="suffix">
            <string/>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelTimeInMs">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelTimeInMsUnit">
           <property name="text">
            <string>ms</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="currentRunnablesList"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labelTotalRunnable">
           <property name="text">
            <string>Total Runnable Count:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelTotalRunnableCount">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="resetModelButton">
       <property name="text">
        <string>Reset Original Values</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelUpperTimeLimit">
       <property name="text">
        <string>Upper time limit:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="timeLimitSpinbox">
       <property name="suffix">
        <string>ms</string>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
